manuscript/manuscript.Rmd

```diff
@@ -491,68 +491,51 @@ Many of our most realistic ecological models exist only as numerical simulations
-
+In seeking either to adopt or assess a deep RL-based approach to an ecological issue, we highlight three key issues that are important to bear in mind: generalizability, interpretability, and reward design.
+
+<!-- reason 1: generalizability-->
+<!-- ML: pivoting this to generalizability but still rewrote past paragraph which might fit in the appendix somewhere-->
+Currently, deep RL agents struggle to generalize their skills [@pmlr-v97-cobbe19a].
+This inability can affect performance during training and at evaluation. 
+For instance, training an agent on an environment that demands learning an assortment of skills is typically much more difficult than training an agent to learn one specific skill [@teh_distral_2017].
+And, at evaluation, if you give an agent a novel task, including tasks that are closely related to the examples that the agent saw in training, the agent can easily fail to generalize its learned behavior to this new situation [@pmlr-v97-cobbe19a].
+These failures are all due to the brittleness of deep neural networks. 
+Across deep learning, neural networks tend to perform exceptionally well on narrowly defined problems, yet neural networks show great fragility when given new situations or problem statements that require flexibility [@cogswell_reducing_2016].
+The ability to adapt to new situations is critical for any decision-making problem in the real world, so the current lack of this ability in deep RL may limit deep RL's effectiveness to conservation problems.
 
 <!--
 Under some conditions, RL agents can prove adept at _transfer learning_, in which an agent trained in one environment performs well under a different environment.
 -->
 <!-- if we're really recommending 'testing' for stability, maybe we need to demo that in the appendix-->
-
 <!-- reason 2: interpretability -->
-<!-- ML: Re our OU talk, might be worth saying that this is a barrier to adoption? If we want to ultimately use these RL agents to make important decisions, it would be hard to swallow us blindly using them without having any sense to what is going on. -->
-Interpretability is a common challenge at the adoption of most ML methods [@cite]. 
-Although there have been some attempts to increase the transparency of RL agents that use neural network as function approximators [cite], it is still very difficult to explain the behavior of RL-trained agents.
-As our examples have illustrated, it may be possible to gain intuition on an agent's behavior by visualizing the action selected against each possible state, though this becomes more difficult in large problems.
-Another challenge is that Deep RL algorithms are very sample inefficient, all though progress is being made to improve this [@cite]. <!-- ML: I think we can cut this sentence, mentionned it in passing earlier-->
--->
-
+Deep RL agents are also difficult to interpret [@heuillet_explainability_2021].
```

Comment:
This is definitely a good issue to discuss, but our examples don't illustrate it so well because we have 1-D action and 1-D state spaces where it's actually quite easy to visualize the policy and not all that hard to 'interpret' the policy.  It seems a bit incongruent to present some semi-interpretable examples.

Reply:
For our models, yes interpretable but not explainable -- like it would still be hard to point inside the network and say what neuron is doing what. I'll probably just use 'transparency' to introduce.